translations/messages+intl-icu.de.yml

```
@@ -3360,6 +3360,7 @@ statement.metadata: "Metadaten"
 error.statement.missing.segment.drafts: "Bitte teilen Sie die Stellungnahme zuerst in Abschnitte auf. Erst dann können Sie das Aufteilen abschließen."
 statement.modal.step.write: "Verfassen Sie Ihre Stellungnahme und geben Sie Metadaten wie Verortung oder Dokumentenbezug an."
 statement.modal.step.personal.data: "Wählen Sie zwischen anonymer und namentlicher Einreichung und geben ggf. Ihre Kontaktdaten an."
+statement.modal.step.privacy.policy: "Bitte beachten Sie die geltende <a class=\"underline\" href=\"/datenschutz\" target=\"_blank\" rel=\"noopener noreferrer\">Datenschutzerklärung</a>."
```

the keys naming actually follows the 3 steps "write", "personal.data" (which should be renamed "personal_data") and "recheck". When adding a key to step 0 [!!!], i'd suggest to append the significant part to the "write" part. also, snake_case because there is no hierarchy expressed. See https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/best-practices/translation-keys.html#use-dots-and-snake-case

```suggestion
statement.modal.step.write.privacy_policy: "Bitte beachten Sie die geltende <a class=\"underline\" href=\"/datenschutz\" target=\"_blank\" rel=\"noopener noreferrer\">Datenschutzerklärung</a>."
```

---

Done in 98f7339c61cbe8900fd4dcb5822733e6f94194a2 and 10e2b9d82752cf8c53b661568b1c6cbd35a21eb9